src/civetweb.c

```diff
@@ -6070,25 +6106,7 @@ push_inner(struct mg_context *ctx,
 		/* Only in case n=0 (timeout), repeat calling the write function */
 
 		/* If send failed, wait before retry */
-		if (fp != NULL) {
-			/* For files, just wait a fixed time.
-			 * Maybe it helps, maybe not. */
-			mg_sleep(5);
-		} else {
-			/* For sockets, wait for the socket using poll */
-			struct mg_pollfd pfd[1];
-			int pollres;
-
-			pfd[0].fd = sock;
-			pfd[0].events = POLLOUT;
-			pollres = mg_poll(pfd, 1, (int)(ms_wait), &(ctx->stop_flag));
-			if (!STOP_FLAG_IS_ZERO(&ctx->stop_flag)) {
-				return -2;
-			}
-			if (pollres > 0) {
-				continue;
-			}
-		}
+		mg_sleep(5);
```

Comment:
I think now poll works (somehow), so we should stick to it

Reply:
This is a change by michael i haven't challenged since I assumed it necessary. I will have a look.